web/src/message_edit.js

```diff
@@ -1044,13 +1045,15 @@ export function save_message_row_edit($row) {
         message = echo.edit_locally(message, currently_echoing_messages.get(message_id));
 
         $row = message_lists.current.get_row(message_id);
+        $(document).trigger("message_editing_done.zulip");
         end_message_row_edit($row);
     }
 
     channel.patch({
         url: "/json/messages/" + message.id,
         data: request,
         success() {
+            $(document).trigger("message_editing_done.zulip");
```

Comment:
What's the thinking around using jQuery events for this? In general we have not added any new ones in years because they have unpredictable behavior and we plan to deprecate jQuery itself eventually.

It's also just awkward that this doesn't pass the `ID` of the message being edited as a parameter, which is clearly important for handlers to correct manage this. Why not just have `message_edit.js` call a function in the `typing*` modules directly?

Reply:
Regarding the passing of `message_id` as a parameter -- I guess if we want editing notification of only one single message at a time (if while editing a message, you move on to editing another message, then the edit notification of first message should be cancelled.), then I guess the `message_id` wouldn't be needed inside the method.

Reply:
updated!